docs/kube-up.md

```diff
@@ -0,0 +1,22 @@
+# Production Quality Cluster on GCE
+This document provides the steps to bring up a production quality cluster on GCE with [`kube-up.sh`](https://kubernetes.io/docs/getting-started-guides/gce/).
+
+## Download CRI-Containerd Release Tarball
+To download release tarball, see [step 1](./installation.md#step-1-download-cri-containerd-release-tarball) in installation.md.
+
+Unpack release tarball to any directory, using `${CRI_CONTAINERD_PATH}` to indicate the directory in the doc:
+```shell
+tar -C ${CRI_CONTAINERD_PATH} -xzf cri-containerd-${VERSION}.linux-amd64.tar.gz
```

Comment:
$

Reply:
ditto.

Reply:
and again consistency.. #, $, vs none for command line examples.. IMO you pick one. I don't care which.

Reply:
Changed all unnecessary `shell` and `console` to `bash` to make it consistent with `README.md`.

For those we want to show the command result, keeps `$` and `console`.